Tasks/XcodeV5/task.loc.json

```diff
@@ -286,7 +286,7 @@
       "name": "destinationSimulators",
       "label": "ms-resource:loc.input.label.destinationSimulators",
       "type": "string",
-      "defaultValue": "iPhone 7",
+      "defaultValue": "",
```

Comment:
We could break users with self-hosted agent, Xcode lower than 11 and not set destinationSimulators by this change

Reply:
Yes, it is supported in xcode 9/10/11 but doesn't seem to be supported in xcode 8

Reply:
I've added the condition for the Xcode 8. Now all available Xcode version options are supported and the default value works correctly.